### DISCUSSION TITLE
How can I update the session object?

### DISCUSSION THREAD

**Discussion by miguelakira** [Hebilicious/authjs-nuxt#123] [Q&A]
I tried using `updateSession` to update the session with an arbitrary data. It works, but when I call the session object again somewhere else, it no longer contains the updated data.

```
const { session, updateSession } = useAuth()
await updateSession((session: Session) => {
    return { ...session, newData: 'something' };
});
```
and somewhere else, `const { session } = useAuth()` will give me the session without newData

**Comment by Hebilicious** [comment]
@miguelakira I created an issue to track this #127 . Could you provide a minimal reproduction to speed things up please ?
